.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: node_js
 node_js:
-  - "0.8"
-  - "0.10"
\ No newline at end of file
+  - "node"
+  - "iojs"
```

Comment:
No worries. It's nice to talk about these details.
I think there are 2 reasonable solutions here...

1. We use an older version of chalk. We could update it to the most recent version that still works on the version of npm that shipped with `node 0.8`.
2. We don't worry about it and ship it as a major release that targets `npm >= 2.whatever` in the engines field of the `package.json`.

`asciify` was written for the fun of it, a long time ago. It's since become depended on by a bunch of projects that want fancy text in there clis. As such, whatever we do, it'd be wise to release this as a major change, to avoid suprising anyone whose running an old app and for whatever reason is stuck on `node 0.8`. No one wants their fancy text dependency to break their build.

I don't think it's worth the hassle of configuring travis to test on `node 0.8` and `0.10` even if it could work there with the right `npm`. Given those versions are both _end of life_ now, it'd be clearer to say that asciify either support them totally, or not at all.

Which do you think? Old chalk and support `node 0.8`, or new chalk and target `node >= 0.12`?